### PR TITLE
fixing menu not working at all

### PR DESCRIPTION
**Pull request by chrisnojima**
@keybase/react-hackers 

the open folders link got broken when we started using the resolve() calls
the open private/public buttons got broken because they weren't flow checked and they were using invalid icons

### CODE REVIEW

**Comment by chromakode** [comment]
:+1: This was a little hard to parse with multiple changes going on in the same diff, but I figured it out!

**Comment by chrisnojima** [comment]
@chromakode sorry, mostly a side-effect of the missing flow annotations which caused the bug in the first place
